web_accounts_list.json

```diff
@@ -2161,6 +2172,17 @@
         "category" : "social",
         "valid" : true
       },
+      {
+        "name" : "Sploder",
+        "check_uri" : "https://www.sploder.com/games/members/{account}",

```

Comment:
I'm getting errors  on this site for the detection strings. Can you check and try again?

` !  ERROR: {'name': 'Sploder', 'check_uri': 'https://www.sploder.com/games/members/{account}', 'account_existence_code': '200', 'account_existence_string': 'Awesomeness', 'account_missing_code': '404', 'account_missing_string': 'The game you were looking for by', 'known_accounts': ['vaxen2', 'geoff'], 'category': 'gaming', 'valid': True} BAD CODE AND STRING. Neither the HTTP response code or detection string worked.`

Reply:
Sploder was acting rather strange, so I just removed it altogether.